src/Workspaces/VisualBasic/Portable/Formatting/VisualBasicSyntaxFormattingService.vb

```diff
@@ -62,4 +46,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
             Return New VisualBasicFormatEngine(root, options, formattingRules, startToken, endToken).Format(cancellationToken)
         End Function
     End Class
+
+#If Not CODE_STYLE Then
+    <ExportLanguageService(GetType(ISyntaxFormattingService), LanguageNames.VisualBasic), [Shared]>
+    Friend Class VisualBasicSyntaxFormattingService
```

Comment:
Move to VB Workspaces layer.

Reply:
yes.  that makes sense.  will do in followup!

Reply:
Done in https://github.com/dotnet/roslyn/pull/60462.